docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -36,17 +38,41 @@ C:\> sc query aesmservice
 The state of the service should be "running" (4). See the Troubleshooting
 section on how to manually install the PSW if it is not installed.
```

Comment:
We should actually remove the troubleshooting section. I don't think that the driver/PSW referenced there is current anymore, and there is a more official version that's up on [Intel's download center.](https://registrationcenter.intel.com/en/products/download/3406/)